data/transactions/logic/assembler.go

```diff
@@ -1516,6 +1542,33 @@ func typePushInts(pgm *ProgramKnowledge, args []string) (StackTypes, StackTypes,
 	return nil, types, nil
 }
 
+func typePushInt(pgm *ProgramKnowledge, args []string) (StackTypes, StackTypes, error) {
+	types := make(StackTypes, len(args))
+	for i := range types {
+		val, err := strconv.ParseUint(args[i], 10, 64)
+		if err != nil {
+			types[i] = StackUint64
+		} else {
+			types[i] = NewStackType(avmUint64, bound(val, val))
+		}
+	}
+	return nil, types, nil
+}
+
+func typeBzero(pgm *ProgramKnowledge, args []string) (StackTypes, StackTypes, error) {
+	// Bzero should only allow its input int to be up to maxStringSize bytes
```

Comment:
Interesting check: we could do that in many places. But maybe not until I clean up the complexity around defining `type` functions.